create-publication-from-scopus/src/test/resources/2-s2.0-0016778795.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Unless the xml files have the word "artificial" in them, they are genuin untampered sample xml files found AWS S3 bucket.